.github/workflows/ci.yaml

```diff
@@ -177,7 +132,7 @@ jobs:
         with:
           cache-from: type=local,src=${{ env.DOCKER_CACHE}}/tests
           cache-to: type=local,dest=${{ env.DOCKER_CACHE}}/tests,mode=max
-          file: IntegrationTests.Dockerfile
+          file: BrowserTests.Dockerfile
           context: .
 
       - name: Run browser tests
```

Comment:
Why are the logs so long for this step? Could we make sure we don't have to scroll that much in the workflows step logs before seeing the tests results?

Reply:
They're the same warnings as before I think. See this previous [integration-tests](https://github.com/mozilla/remote-settings/actions/runs/8047020616/job/21975436899#step:9:978) run and this previous [browser-tests](https://github.com/mozilla/remote-settings/actions/runs/8047020616/job/21975437573#step:8:991) run. Five `DeprecationWarning`s from autograph_utils, one from kinto_http, and a config warning about `sensitive_url`.

I could suppress the warnings but that seems like a bad idea. We should probably get updating autograph and kinto_http on the radar.

Reply:
Added `-q` to `docker compose build` to quiet that part down at least.